lib/db.js

```diff
@@ -790,6 +791,50 @@ module.exports = function (
     return this.pool.get('/securityEvents/' + params.uid.toString('hex') + '/ip/' + params.ipAddr)
   }
 
+  DB.prototype.createUnblockCode = function (uid) {
+    log.trace({
+      op: 'DB.createUnblockCode',
+      uid: uid
+    })
+    var self = this
+    var unblock = UnblockCode()
+    return this.pool.put('/account/' + uid.toString('hex') + '/unblock/' + unblock)
```

Comment:
`unblock` is always a String, since being base36 makes it inconvenient to be represented in binary (every conversion back to a String is a possible failure, that we'd probably say "cannot happen because we only ever convert base36 -> binary", but programmers always make mistakes).